<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEmployeeView</class>
 <widget class="QMainWindow" name="AddEmployeeView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="firstname">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Employee First Name</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="lastname">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>40</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Employee Last Name</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="username">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Employee Username</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="password">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>90</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Employee Password</string>
    </property>
   </widget>
   <widget class="QComboBox" name="isActive">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>103</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Active</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Inactive</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="privledgeLevel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>150</y>
      <width>103</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Owner</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Manager</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Supervisor</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cashier</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Inventory</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>General</string>
     </property>
    </item>
   </widget>
   <widget class="QTextEdit" name="payRate">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Pay Rate</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="address">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>190</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Address</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addEmployee">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>250</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Employee</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>300</y>
      <width>551</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Data will appear here</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
